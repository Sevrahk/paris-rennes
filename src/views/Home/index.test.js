import React from 'react';
import { shallow } from 'enzyme'
import Component from './index';

describe('Stars component', () => {
    let component;

    beforeEach(() => {
        component = shallow(<Component />);
    });

    test('renders', () => {
        expect(component.find('Catalog')).toHaveLength(1);
    });

    test('preserve UI', () => {
        expect(component).toMatchSnapshot();
    });
});
