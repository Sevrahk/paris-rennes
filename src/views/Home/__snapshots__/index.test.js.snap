// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stars component preserve UI 1`] = `
<Fragment>
  <header>
    <h1>
      De Paris Ã  Rennes
    </h1>
  </header>
  <Catalog
    apiToken={null}
  />
</Fragment>
`;
